### DISCUSSION TITLE
2.1.0rc2

### DISCUSSION THREAD

**Discussion by eschutho** [apache/superset#23357] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 2.1.0.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/2.1.0rc2/

Git tag for the release:
https://github.com/apache/superset/tree/2.1.0rc2

The Change Log for the release:
https://github.com/apache/superset/blob/2.1.0rc2/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/2.1.0rc2/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

Please vote according to the voting thread.

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/2.1.0rc2'>2.1.0rc2</a>.</em>

**Comment by hbruch** [comment]
I suppose it's too late for release 2.1.0, but for upcoming releases, I'd suggest that updating the messages.pot and - volunteers for translating new and updated messages assumed - the messages.po files become steps of the release check list. A good point in time might be just before forking the release branch from the master branch(?).
 
I appreciate very much the progress superset has made again and am happy to contribute back translating (in my case to German), so that non-English-speakers have a good user experience as well. Thanks everybody for the incredible work!

**Reply by hbruch** [reply]
In case there would be a 2.1.0rc3, you might consider PR #23402 which provides missing localized German messages.

**Comment by sfirke** [comment]
I tested rc2 today, things seem pretty good.  The tabbed-dashboards-not-loading bug is fixed which is great.

I filed some bugs that I believe are unrelated to 2.1.0rc2 specifically, won't mention those here.  In terms of regressions affecting 2.1.0rc2, for me it's these two minor ones:
- #23197 is a minor regression from 2.0.1.  Has anyone looked at it to see if there's an easy fix?  It's not a show-stopper.
- #23312 people are having a hard time replicating this (which means there's not an obvious fix) and I can work around it by using Chrome

**Reply by sfirke** [reply]
Update: I can corroborate the report from the dev email list that Alerts & Reports are broken in 2.1.0rc2.  I'm experiencing this: https://github.com/apache/superset/issues/22326.  I tried specifying `WEBDRIVER_OPTION_ARGS = ["--headless"]` in my config but that just changed the failure error message to `Process unexpectedly closed with status 255`.

**Reply by nytai** [reply]
This issue should be fixed with: https://github.com/apache/superset/pull/23388

**Comment by squalou** [comment]
Testing it right now, seen lots of nice features and fixes of annoying bugs.

May I suggest this (patched for my personal use but who knows), minor issue with Excel export. #23375

**Reply by eschutho** [reply]
Thank you @squalou, this is helpful information. We can try to get a fix into the next release. If anyone from the community is interested in helping with a PR, that would help push this along.

**Reply by squalou** [reply]
juste to illustrate further : I did a fork.

https://github.com/squalou/superset/commit/f2768e6f70778388f9f18a76ac8ab9b6cc14869d


 I'm pretty sure using an array an relying on column order is not the safest solution, but I'm a python noob and don't have superset's big picture in mind, so I'l let the experts implement a proper fix :-)

**Comment by amartincolville** [comment]
hello! 
We are also doing some rc2 testing and the new feature for report sharing, for example, are great! And some bug fixes are looking fit.
We are seing, nevertheless, some issue regarding the loading of some resources, I believe this refers to slices:
`Failed to load resource: the server responded with a status of 403 ()` in the console when entering the homepage, with a Forbidden warning.

![Screenshot 2023-03-21 at 13 23 58](https://user-images.githubusercontent.com/91057381/226605575-8e651bac-d054-4f6e-97fc-91ce561ade20.png)

This applies to datasets I believe, too, as although it already exists, we are seeing the following on Charts that were already created.

![Screenshot 2023-03-21 at 13 24 20](https://user-images.githubusercontent.com/91057381/226605715-e02ee656-7064-49b3-b406-42e2e58cb052.png)
 
Thus, charts are not loading.

All this is being done from then Admin role and I have seen the endpoint migrations (which might be driving this) although the ones to switch to not always exist in the permissions e.g. for changing the corresponding permissions from `can filter on Superset` to `can get column values on Datasource`, the first one is still in the Role whilst the latter is not found. 
I am not sure if this is the direct cause of the issue although it might have an impact.
Any thoughts on this? Also, happy to provide details or change anything if I maybe missed out on anything, thank you in advance!

**Reply by amartincolville** [reply]
we found the issue and migrations were not applied correctly on our side. Nevertheless, we have spotted some inconsistencies in `UPDATING.md` - good for `rc3`!
